package.json

```diff
@@ -45,6 +45,7 @@
     "hast-util-is-element": "^2.0.0",
     "hast-util-to-text": "^3.0.0",
     "mdast-util-phrasing": "^3.0.0",
+    "hast-util-phrasing": "^2.0.1",
```

Comment:
In unified, we prefer to keep loose ranges, if possible, so `2.0.0`. It helps resolving packages by making the ranges loose. And it keeps package lock diffs cleaner.
I’ll fix this myself here, but just an FYI for the future!

Reply:
Ok, thanks. I'm learning as I go along with npm, package.json as well as git.

Reply:
no problem, I’ll try to keep that mind, and do let me know when there’s something you’re wondering about in all this!